jwt.go

```diff
@@ -384,6 +395,44 @@ func (jwtPlugin *JwtPlugin) ExtractToken(request *http.Request) (*JWT, error) {
 	return &jwtToken, nil
 }
 
+func (jwtPlugin *JwtPlugin) remoteAddr(req *http.Request) Network {
+	// This will only be defined when site is accessed via non-anonymous proxy
+	// and takes precedence over RemoteAddr
+	// Header.Get is case-insensitive
+	ipHeader := req.Header.Get("X-Forwarded-For")
+	if len(ipHeader) == 0 {
+		ipHeader = req.RemoteAddr
+	}
+
+	ip, port, err := net.SplitHostPort(ipHeader)
+	portNumber, _ := strconv.Atoi(port)
```

Comment:
We could also make the port a string, given that it is only used for logging anyway.

Reply:
Does DataDog interpret them differently?